gluon/custom_import.py

```diff
@@ -67,6 +67,13 @@ def custom_importer(name, globals=None, locals=None, fromlist=None, level=-1):
         import_tb = None
         try:
             try:
+                if ('gluon' not in name and not name.startswith('.') and
```

Comment:
What do you propose to fix the mentioned issues?

 Paolo

2015-01-23 21:25 GMT+01:00 Diogo <notifications@github.com>:

> In gluon/custom_import.py
> <https://github.com/web2py/web2py/pull/587#discussion_r23477626>:
>
> > @@ -67,6 +67,13 @@ def custom_importer(name, globals=None, locals=None, fromlist=None, level=-1):
> >          import_tb = None
> >          try:
> >              try:
> > +                if ('gluon' not in name and not name.startswith('.') and
>
> +1
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/web2py/web2py/pull/587/files#r23477626>.
>